### DISCUSSION TITLE
Filter on unknown Source

### DISCUSSION THREAD

**Discussion by danieljbradley** [plausible/analytics#1397] [Anything Else]
I want to be able to look at metrics for users with unknown Source.  As an example, I had 7 visitors today.  When I look at source I see 5 from Google Ads and 1 from DuckDuckGo.  The 7th visitor is not shown in the source table.

If I want to see duration and page count metrics, for Google Ads visitors, I can filter on that but it doesn't seem to be possible if the source is not listed.  (I know I can use "is not google" but the "is not" won't be useful if there is more than 1 known source.  Does anybody know how to do this?  Thank you

**Comment by metmarkosaric** [comment]
If you click on "Details" in your Top Sources report, you'll see "Direct/None" in the list of referral sources.

**Reply by danieljbradley** [reply]
Beautiful!  Thank you
